### PR TITLE
Readme fixes

### PR DESCRIPTION
**Pull request by mbuttler**

### CODE REVIEW

**Review by haggaret — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,31 +1,32 @@
-#Umpire
+# Umpire Dependency Manager
```
Comment (miketypeguy) [inline README.md]: I mean the name is Umpire and it is a dependency manager but I wouldn't say that has to be part of the name.

File: README.md [hunk 2]
```diff
@@ -1,31 +1,32 @@
...
+
+Umpire is available as a [pip package]((https://pypi.python.org/pypi/pip), and requires Python 2.7.15.
+
+To install Umpire from the command line:
 
 ```
 $ pip install umpire
 ```
 
-Don't have pip? Get it [here.](https://pypi.python.org/pypi/pip)
```
Comment (miketypeguy) [inline README.md:16]: I don't see anything wrong with mentioning how to get pip

File: README.md [hunk 3]
```diff
@@ -1,31 +1,32 @@
...
+# Umpire Dependency Manager
 
-Umpire is a generic dependency resolver which aims to be both platform agnostic, and downright simple to use. It is developed in Python and is easily extensible.
+Umpire is a Python-based generic dependency resolver which aims to be platform agnostic, extensible, and simple to use.
 
-Using an S3 backed repository of compressed packages, Umpire reads a simple JSON file to retrieve, cache, and link files to their appropriate destination. It's original use case was to keep binary artifacts and tools out of git repositories, but still provide a way for these binaries to be versioned.
+Umpire was developed as an easy to install command line tool, with it's flexibility being derived from the JSON file options.
 
-Umpire is being developed as an easy to install command line tool, with it's flexibility being derived from the JSON file options.
+Umpire reads a JSON file to retrieve, cache, and link files to their appropriate destination from an Amazon S3 backed repository of compressed packages.
```
Comment (miketypeguy) [inline README.md:7]: The way this was switched around makes sense and still relays the pertinent information.

File: README.md [hunk 4]
```diff
@@ -52,20 +53,20 @@ There's also a couple of other options that you can specify. They are:
  ]
 ```
 
-### Development
+## Contributing
 
-Want to help out? Awesome!
+Want to help out?
```
Comment (miketypeguy) [inline README.md]: I think helping out is Awesome still.

**Review by miketypeguy — CHANGES_REQUESTED** [review]
Take a peak at some of my comments and let me know why I'm wrong 👍

File: README.md (hunk 2 above)
Comment (mbuttler) [reply README.md:16]: Agreed. Take thyself a look.

**Review by miketypeguy — APPROVED** [review]
Looks good

**Pull request merged**